utils/ui.js

```diff
@@ -8,3 +9,17 @@ export const linkCn = ({ className, disabled, dark, active } = {}) =>
     { 'wir-link--disabled': disabled, 'wir-link--theme-dark': dark, 'wir-link--active': active },
     className
   );
+
+// https://stackoverflow.com/questions/5623838/rgb-to-hex-and-hex-to-rgb
+const hexToRgb = hex =>
+  hex
+    .replace(/^#?([a-f\d])([a-f\d])([a-f\d])$/i, (m, r, g, b) => `#${r}${r}${g}${g}${b}${b}`)
```

Comment:
Please extract regex to constant & add link to https://regex101.com

Reply:
for example: https://github.com/babajka/babajka-backend/blob/develop/src/services/fibery/utils.js#L14-L15